sdk/src/utils/serde.rs

```diff
@@ -131,6 +131,33 @@ pub mod string_prefix {
     }
 }
 
+pub mod option_string_prefix {
```

Comment:
Just wondering, is there no shorter of such things when we already have the non-option version?

Reply:
Cool, my question is still valid for the other ones if ever you have an answer

Reply:
I have not found a shorter way